src/configuration/Configuration.container.tsx

```diff
@@ -28,6 +29,12 @@ const ConfigurationContainer: FC<ConfigurationContainerProps> = ({
       )
     : null;
 
+  if (!configurationSettings?.spec?.deploymentPlan.image) {
```

Comment:
We want to test if the `deploymentPlan` exists, use this instead:
`if (!configurationSettings?.spec?.deploymentPlan) {`

Reply:
yes, i will try that once i fix the error with plugin running